### ISSUE TITLE
Video recording doesn't work on live branch (full suite) tests

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-e2e-tests#1399]
See: https://circleci.com/gh/Automattic/wp-e2e-tests-for-branches/12812

@bsessions85: I've temporarily disabled it via an env variable to stop the errors, do the wrapper repos need more dependencies installed in their config? What about other repos like Jetpack?

**Comment by bsessions85** [comment]
@alisterscott  yes, I forgot that they need to install ffmpeg. I have updated wp-e2e-tests-for-branches and it is working now. I have PRs in to jetpack and jetpack-be wrapper repos that I will get merged in when I can get all the tests to pass.
